index.html

```diff
@@ -0,0 +1,44 @@
+<link rel="stylesheet" href="stylesTributePage.css">
+
+<style>
+   body {
+      background-color: rgb(90, 0, 80);
+      color: white;
+      font-family: calibri;
+   }
+</style>
+
+<html lang="en">
+
+<head>
+   <title>Tribute Page</title>
+</head>
+
+<body>
+
+   <div id="main">
+      <h1 id="title">Sevagoth - The slayer</h1>
+
+      <div id="img-div">
+         <a id="tribute-link" target="_blank" href="https://warframe.fandom.com/wiki/Sevagoth">
+            <img id="image"
+               src="https://www.ggrecon.com/media/x4tlnglf/warframe-sevagoth-2.jpg?mode=crop&width=682&quality=80&format=webp"
+               alt="Super duper cool Sevagoth image" width="100%"></a>
```

Comment:
Using "width" in html and in CSS will cause conflict between each other. Styling should only be done in CSS.

Reply:
- [ ] Dear @Thinus01, Good job providing the feedback with explanation on why. You can Improve this further by making your reviews polite, by giving appreciation for what have been don good so far and adding  `to make it better`  , `Please`,  `kindly`  or other terms that sound suggestion than order. This allows you to sound more friendly in your comment. you could use a format like `good job doing this`, then `pleas or kindly apply this`  then `the reason for your suggestion`